docs/visual-basic/misc/bc30949.md

```diff
@@ -65,6 +65,6 @@ Next i
   
 ## See Also  
  [For Each...Next Statement](../../visual-basic/language-reference/statements/for-each-next-statement.md)   
- [NOTINBUILD Overview of Arrays in Visual Basic](http://msdn.microsoft.com/en-us/ca50e2f2-b4d2-4c57-9169-9abbcc3392d8)   
+ [Arrays in Visual Basic](~/docs/visual-basic/programming-guide/language-features/arrays/index.md)   
  [How to: Initialize an Array Variable in Visual Basic](../../visual-basic/programming-guide/language-features/arrays/how-to-initialize-an-array-variable.md)   
  [NOTINBUILD How to: Initialize a Multidimensional Array](http://msdn.microsoft.com/en-us/502dcf8b-d86c-46f1-ad7d-3ce809645774)
```

Comment:
This should be `[Array dimensions in Visual Basic](~/docs/visual-basic/programming-guide/language-features/arrays/array-dimensions.md) `